### PR TITLE
Release/alpha 0.0.1

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

File: src/NewsService.Data/NewsRepository.cs [hunk 1]
```diff
@@ -1,89 +1,104 @@
...
+
+      if (findNewsFilter.TakeCount < 1)
+      {
+        totalCount = 0;
+        return null;
+      }
+
+      IQueryable<DbNews> dbNewsList = _provider.News.AsQueryable();
+
+      if (findNewsFilter.AuthorId.HasValue)
```
Comment (BelehovEgor) [inline src/NewsService.Data/NewsRepository.cs:79]: move to separate method

File: src/NewsService.Mappers/Db/DbNewsMapper.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+      return new DbNews
+      {
+        Id = Guid.NewGuid(),
+        Preview = request.Preview,
+        Content = request.Content,
+        Subject = request.Subject,
+        Pseudonym = !string.IsNullOrEmpty(request.Pseudonym?.Trim()) ? request.Pseudonym.Trim() : null,
+        AuthorId = request.AuthorId,
+        DepartmentId = departmentId?[0],
```
Comment (BelehovEgor) [inline src/NewsService.Mappers/Db/DbNewsMapper.cs:35]: firstOrDefault?

File: src/NewsService.Mappers/Models/NewsInfoMapper.cs [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+using MassTransit;
+using Microsoft.Extensions.Logging;
+
+namespace LT.DigitalOffice.NewsService.Mappers.Models
+{
+  public class NewsInfoMapper : INewsInfoMapper
+  {
+    private readonly IRequestClient<IGetUserDataRequest> _userRequestClient;
+    private readonly IRequestClient<IGetDepartmentRequest> _departmentRequestClient;
+    private readonly ILogger _logger;
```
Comment (BelehovEgor) [inline src/NewsService.Mappers/Models/NewsInfoMapper.cs:19]: add template

**Review by BelehovEgor — APPROVED** [review]

**Review by Cheephot — APPROVED** [review]

**Review by borovHokage — APPROVED** [review]

File: src/NewsService.Data/NewsRepository.cs (hunk 1 above)
Comment (MariyaSin) [reply src/NewsService.Data/NewsRepository.cs:79]: added to Redmine task 872

File: src/NewsService.Mappers/Models/NewsInfoMapper.cs (hunk 3 above)
Comment (MariyaSin) [reply src/NewsService.Mappers/Models/NewsInfoMapper.cs:19]: added to Redmine task 872

**Pull request merged**